### ISSUE TITLE
Flatulence

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6644]
**Preferred term label:**
Flatulence

**Synonyms**
Gas

**Definition (free text, please give PubMed ID)**
Passage of excessive amounts of gas and the feeling of abdominal fullness and bloating.;  PMID: 27275398

**Parent term (use hpo.jax.org/app)**
Abdominal symptom

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 223100

**Comment by pnrobinson** [comment]
added term
